mozilla/bedrock
# [fix bug 1422094] Updates to Mozilla 2016 Annual report

## Description
Minor updates to 2016 State of Mozilla report:
- Make the hero image smaller.
- Place top italic intro paragraph before the first heading.
- Add "Join Mozilla" heading to the last paragraph.
- Insert chart in Maintaining Sustainability section.

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1422094

## Testing
- Check header respnosiveness?
- Check for copy/pasta errors?
